db/warehouse/migrate/20240205230723_add_display_hook_to_custom_data_element_def.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisplayHookToCustomDataElementDef < ActiveRecord::Migration[6.1]
+  def change
+    add_column :CustomDataElementDefinitions, :show_in_summary, :boolean, default: false, null: false, comment: 'Whether to show this custom field in summary views such as in a table row when viewing a Service/CLS/Note'
```

Comment:
Maybe the comment here could go on the graphql field description? I'm unsure of the value of putting documentation in sql comments but maybe I just don't have a good workflow for discovering them.

Reply:
Yeah I'll put the comment in gql. The col descriptions show up in TablePlus which I sometimes find helpful, but they aren't maintained, so it's not the best thing.